### PR TITLE
281_add_int_test

### PR DESCRIPTION
**Pull request by pawelszymczyk**
I added integration test for check that after rollback topic still exists on broker.

### CODE REVIEW

File: integration/src/integration/java/pl/allegro/tech/hermes/integration/management/TopicManagementTest.java [hunk 1]
```diff
@@ -209,4 +209,22 @@ public void shouldCreateTopicEvenIfExistsInBrokers() {
         assertThat(response).hasStatus(Response.Status.CREATED);
         Assertions.assertThat(management.topic().get(qualifiedTopicName)).isNotNull();
     }
+
+    @Test
+    public void topicCreationRollbackShouldNotDeleteTopicOnBroker() throws Throwable {
+        // given
+        String groupName = "existingTopicFromExternalBroke22r";
```
Comment (adamdubiel) [inline integration/src/integration/java/pl/allegro/tech/hermes/integration/management/TopicManagementTest.java]: 22r? :)

**Comment by adamdubiel** [comment]
:+1:

File: integration/src/integration/java/pl/allegro/tech/hermes/integration/management/TopicManagementTest.java (hunk 1 above)
Comment (pawelszymczyk) [inline integration/src/integration/java/pl/allegro/tech/hermes/integration/management/TopicManagementTest.java]: Sry for that, now it's renamed for: "topicCreationRollbackShouldNotDeleteTopicOnBroker"

**Comment by pbetkier** [comment]
:+1:

**Pull request merged**